src/core/useAtom.ts

```diff
@@ -59,7 +61,14 @@ export function useAtom<Value, Update, Result extends void | Promise<void>>(
     [store, atom]
   )
 
-  const [[version, value, atomFromUseReducer], dispatch] = useReducer<
+  // This reducer stores this hook's view of the atom's state (prev[1]).
+  //
+  // We know the component should re-render whenever it's view of the atom changes.
+  //
+  // If we did change tracking using a previous state stored outside of React,
+  // that state could be out of sync with React's concurrent rendering with
+  // `useTransition`.
```

Comment:
I'm not sure if I understand what this note is trying to say.

Reply:
I decided to remove this one, since I'm not sure either 😓

Reply:
okay!